### ISSUE TITLE
Statusanzeige in der WebUI (FHEM) fehlerhaft

### ISSUE THREAD

**Issue by hatage** [delMar43/FHEM#16]
Ich nutze die Version 70_DENON_AVR.pm:0.257870/2022-03-06.
Es wird in FHEM der Status "disconnected" angezeigt, obwohl sich mein Denon AVR X3200 über FHEM steuern lässt.

Als Reading "model" wird mir eine fehlerhafte Darstellung angezeigt (ohne die Anführungszeichen):
"< html>< head><title>Document Error: Site or Page Not Found< /title>< /head>
		< body>< h2>Access Error: Site or Page Not Found< /h2>
		< p>Cannot open URL< /p>< /body>< /html>"
Anmerkung: die Leerzeichen nach den "<" habe ich wegen der Darstellung eingefügt.

Ich vermute, dass es mit dem letzten Firmwareupdate des AVR zusammenhängt. Diese lautet 5700-0591-4423.
FHEM und das Modul sind bei mir aktuell (fhem.pl 25896 2022-03-30 07:18:07Z).

VG

**Comment by delMar43** [comment]
Es gibt jetzt ein neues Attribut: DeviceInfoPort
Versuch mal, das auf 8080 zu setzen.

**Comment by hatage** [comment]
Danke für die Antwort.
Habe ich gemacht, der Fehler bleibt (auch nach einem Reload).

**Comment by delMar43** [comment]
Alles klar, danke fürs checken.  ich muss mir das ansehen.

**Comment by hatage** [comment]
Gibt es schon etwas Neues zum issue?

**Comment by delMar43** [comment]
Nein, ich kann das Problem auch nicht nachstellen, weil mein AVR schon lange keine Firmware Updates mehr kriegt.
Das Steuern funktioniert aber einwandfrei, richtig?
Ich kann sonst einfach ein Attribut einbauen, dass die Prüfung des Models unterdrückt. Wird rein für die Statistik benötigt (https://fhem.de/stats/statistics.html)
